src/App.js

```diff
@@ -53,17 +55,87 @@ const App = () => {
     setResults(workout);
   };
 
+  const [removeExercises, setEx] = useState(null);
+  const removeEx = (workout) => {
+    setEx(workout);
+    setTotal((count) => count - 1);
+
+    const exerciseIndex = selectedExercises.findIndex(
+      (i) => i.name === workout.name
+    );
+    selectedExercises.splice(exerciseIndex, 1);
+
+    console.log("Exercise: " + workout.name);
+    console.log("Index: " + exerciseIndex);
+  };
+
+  const toggleDrawer = (anchor, open) => (event) => {
```

Comment:
What is this function doing? From what I understand, if a user clicks and has the Tab key or Shift key down as well, then it won't do anything, otherwise, open the drawer?

Reply:
I would honestly like to understand this a little bit more myself. This was in the Material UI documentation for the Drawer component. 

I think the biggest takeaway is the... setDrawer({ ...drawer, [anchor]: open });
We could probably remove the if statement if it's just unnecessary code. 

What do you think?

Reply:
I'll have a look after work and see if I can make sense of what is going on.

Reply:
Sounds good! I will take a look at it as well.